transform-service/latest/install/index.md

```diff
@@ -269,7 +269,7 @@ from the left column that corresponds to the required Content Services version y
     ```
 
     > **Note:** Replace the version number `x.y.z` with the tag that matches the Content Services version you want to 
-    > deploy. For example, if you want Content Services 7.3.0, then select tag `7.3.0`.
+    > deploy. For example, if you want Content Services 7.3.0, then select tag `5.3.0`.
```

Comment:
Let's change the example to ACS **7.4.0** - tag `6.0.0`

Reply:
Updated in commit [782fa07](https://github.com/Alfresco/docs-alfresco/pull/1017/commits/782fa07d8413e0d9cbcd31bb19c6d5ee8df111bf)